client/app/bundles/shared/components/Tag.jsx

```diff
@@ -0,0 +1,29 @@
+//@flow
+import React from "react";
+import css from "./Tag.scss";
+
+type Props = {
+  dark?: boolean,
+  normal?: boolean,
+  label?: string,
+  id?: string,
+  };
+
+export default class Tag extends React.Component<Props> {
+  constructor(props: Props)
+  {super(props);}
+  render()
+   {
+     const {dark,normal,label,id} = this.props;
+     const labelClassNames = `${css.label} ${dark ? css.dark : ""}${normal ? css.normal : ""}`;
+     const tagClassNames = `${css.tag} ${dark ? css.dark : ""}${normal ? css.normal : ""}`;
+          return (
+        <span>
+          <tag
```

Comment:
This seems removable~. This doesn't match up with any pre-existing components nor is it a valid HTML tag!